script.js

```diff
@@ -1,28 +1,97 @@
-var values  = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A"];
-var suits   = ["Clubs", "Diamonds", "Hearts", "Spades"];
+var vals = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A"];
+var suits = ["Clubs", "Diamonds", "Hearts", "Spades"];
 var game = {
-  deck: [],
-  players: [],
-  hands: [],
-  buildDeck: function(){
-
-  },
-  shuffleDeck: function(){
-
-  },
-  getMorePlayers: function(){
-
-  },
-  deal: function(){
-
-  },
-  findHighestCard: function(){
-
-  },
-  announceWinners: function(){
-
-  },
-  playANewGame: function(){
-
-  }
+    deck: [],
+    players: [],
+    hands: [],
+    buildDeck: function(){
+        this.deck = [];
+        for (var _ = 0; _ < suits.length; _++) {
```

Comment:
I know, I changed it to `val` because there was a global name conflict with `values`.  Every time I tried checking `values` in console it was already defined as some browser specific function.